src/settings.cpp

```diff
@@ -843,13 +849,18 @@ if ( GetFlagIniSet ( IniXMLDocument, "server", "defcentservaddr", bValue ) )
     }
 
     // start minimized on OS start
-    if ( GetFlagIniSet ( IniXMLDocument, "server", "autostartmin", bValue ) )
+    if ( !CommandLineOptions.contains( "--startminimized" ) )
     {
-        pServer->SetAutoRunMinimized ( bValue );
+         if ( GetFlagIniSet ( IniXMLDocument, "server", "autostartmin", bValue ) )
+         {
+             pServer->SetAutoRunMinimized ( bValue );
+         }
     }
 
     // licence type (command line overwrites setting file)
-    if ( pServer->GetLicenceType() == LT_NO_LICENCE )
+    //if ( pServer->GetLicenceType() == LT_NO_LICENCE )
+    // TODO: command line needs to become boolean, at least...
```

Comment:
Why should the command line be a boolean value? It is actually a boolean value. Do you want the command line to _not_ be a boolean value instead?

Reply:
Maybe I misread it -- I thought it was only checking for presence, rather than value.